spec/controllers/released_form_controller_spec.rb

```diff
@@ -28,36 +30,7 @@
           AccountName: "New web test org",
           AccountExternalID: "5188ff34-9011-4458-a9ea-19fe1ac9f3c6",
           released_form: {}
-      }
-
-      expect(project.released_forms.first.form_type)
-          .to eq("permission_to_start")
-
-    end
-
-    it "should not release the permission to start from a webhook with Finance" do
-
-      project = create(
-          :project,
-          id: '2c660111-ab15-4221-98e0-cf0e02748a9c'
-      )
-
-      post :receive, params: {
-          Status: "Awarded",
-          ProjectCosts: ["PC-000095"],
-          forms:[],
-          Email: "lizzie90@example.com",
-          ContactName: "Nelda",
-          ApprovedPurpose: {},
-          ApplicationStages: "Finance Approval",
-          ApplicationId: project.id,
-          AccountName: "New web test org",
-          AccountExternalID: "5188ff34-9011-4458-a9ea-19fe1ac9f3c6",
-          released_form: {},
-          ApprovedbyFinance:true}
-
-      expect(project.released_forms.first.form_type)
-          .to eq("completion_report")
+      } }.to have_enqueued_job
```

Comment:
Could we be more specific about the job being enqueued here?

```suggestion
      } }.to have_enqueued_job(ReleaseFormJob)
```